makefile.shared

```diff
@@ -59,14 +61,14 @@ endef
 
 $(foreach demo, $(strip $(DEMOS)), $(eval $(call DEMO_template,$(demo))))
 
-install: $(call print-help,install,Installs the library, headers and pkd-config file) .common_install
+install: $(call print-help,install,Installs the library$(comma) headers and pkg-config file) .common_install
```

Comment:
I would simply leave out the comma: `$(call print-help,install,Installs the library + headers + pkg-config file)`

Reply:
Done.